msgpack.gemspec

```diff
@@ -15,7 +15,11 @@ Gem::Specification.new do |s|
     s.files = Dir['lib/**/*.rb', 'lib/**/*.jar']
     s.platform = Gem::Platform.new('java')
   else
-    s.files = `git ls-files`.split("\n")
+    s.files = Dir.chdir(__dir__) do
+      `git ls-files -z`.split("\x0").select do |file|
+        file.start_with?('ext', 'lib', 'ChangeLog', 'LICENSE', 'README')
```

Comment:
Hum, might as well use the default exclusion list generated by `bundle init` https://github.com/Shopify/gvltools/blob/3cfa5359cd7fe23a8a7fa0b2416df976a7720cb0/gvltools.gemspec#L24-L28

We can add `doclib` to it.

Inclusion list make me a bit uneasy, as it is easy to forget to add a new directory to the list and publish broken releases.

Reply:
Updated in [`9acbd68`](https://github.com/msgpack/msgpack-ruby/pull/311/commits/9acbd68cec1ec8fca8c2cb44052c1630b7d62015).